.buildkite/scripts/common.sh

```diff
@@ -10,34 +10,45 @@ GITHUB_PR_TRIGGER_COMMENT=${GITHUB_PR_TRIGGER_COMMENT:-""}
 ONLY_DOCS=${ONLY_DOCS:-"true"}
 runLibbeat="$(buildkite-agent meta-data get runLibbeat --default ${runLibbeat:-"false"})"
 runMetricbeat="$(buildkite-agent meta-data get runMetricbeat --default ${runMetricbeat:-"false"})"
+runPacketbeat="$(buildkite-agent meta-data get runPacketbeat --default ${runPacketbeat:-"false"})"
```

Comment:
A little unrelated to this PR, but what we are doing here makes manual reproductions difficult (i.e. this command fails when run manually). Could we improve the conditional such that `buildkite-agent meta-data get` is run only if the env var `runPackagebeat` is undefined or not false? (applies to all declarations here).

For context: https://elastic.slack.com/archives/C0D8SUKB2/p1707380999484699?thread_ts=1707312641.347669&cid=C0D8SUKB2

Reply:
This option is for the future main pipeline, where we will define the steps of the pipeline...
We have a similar variable in the main pipeline - `runAllStages`